### ISSUE TITLE
Lets Encrypt: no

### ISSUE THREAD

**Issue by R0GGER** [switnet-ltd/quick-jibri-installer#28]
My problem **From top to bottom**:

```
> Do you plan to use Let's Encrypt SSL certs?: (yes or no)
no
We'll let you choose later on for it.
> Please set your domain (or subdmain) here: (e.g.: jitsi.domain.com)
recorder.domain.nl
> Did you mean?: recorder.domain.nl (yes or no)
yes
Alright, let's use recorder.domain.nl.
Server public IP (my-ip-address) & DNS record for recorder.domain.nl don't seem to match.
Please check your dns records are applied and updated. Exiting for now...
```
Do you plan to use Let's Encrypt SSL certs?: (yes or no)
**no**

--- 
IP set in DNS, next try.

Settings:
lets encrypt: **no**
domain: recorder.domain.nl
Certificate: selfsigned certificate
Sysadmin email: mail@domain.nl
Language: en
Translate 'Participant': en
Drop support for unsecure protocols TLSv1.0/1.1: yes
Setup LetsEncrypt with your domain: **no** (Again? I allready choose no.)
Install customized "brandless mode": no (not sure what this is..?)
Welcome page: no
Jitsi Meet Auth Method: 1 (none)
Jibri Records Access via Nextcloud: no
Jigasi Transcription: no
Grafana Dashboard: no
Set username for secure room moderator: administrator
Secure room moderator password: superuberpassword!#

```
Secure rooms are being enabled...
You'll be able to login Secure Room chat with 'administrator' or 'administrator@recorder.domain.nl' using the password you just entered.
If you have issues with the password refer to your sysadmin.
# Checking /etc/jitsi/meet/recorder.domain.nl-config.js file for errors
# The /etc/jitsi/meet/recorder.domain.nl-config.js configuration seems correct. =)
Created symlink /etc/systemd/system/multi-user.target.wants/jibri.service -> /etc/systemd/system/jibri.service.
Created symlink /etc/systemd/system/jibri.service.wants/jibri-xorg.service -> /etc/systemd/system/jibri-xorg.service.
Created symlink /etc/systemd/system/jibri-icewm.service.wants/jibri-xorg.service -> /etc/systemd/system/jibri-xorg.service.
Created symlink /etc/systemd/system/jibri.service.wants/jibri-icewm.service -> /etc/systemd/system/jibri-icewm.service.
SSL setup will be skipped.
Waiting prosody restart to continue configuration, 15s...
Saving debug log to /var/log/letsencrypt/letsencrypt.log
Plugins selected: Authenticator standalone, Installer None
Skipped user interaction because Certbot doesn't appear to be running in a terminal. You should probably include --non-interactive or --force-interactive on the command line.
Obtaining a new certificate
Performing the following challenges:
http-01 challenge for recorder.domain.nl
Waiting for verification...
Cleaning up challenges
Failed authorization procedure. recorder.domain.nl (http-01): urn:ietf:params:acme:error:unauthorized :: The client lacks sufficient authorization :: Invalid response from https://domain.net/page-not-found [my-ip-address]: "<!DOCTYPE html>\n<html lang=\"en\">\n<head>\n<meta charset=\"UTF-8\">\n<meta name=\"viewport\" content=\"width=device-width, initial-scale="
IMPORTANT NOTES:
 - The following errors were reported by the server:

   Domain: recorder.domain.nl
   Type:   unauthorized
   Detail: Invalid response from https://domain.net/page-not-found
   [my-ip-address]: "<!DOCTYPE html>\n<html
   lang=\"en\">\n<head>\n<meta charset=\"UTF-8\">\n<meta
   name=\"viewport\" content=\"width=device-width, initial-scale="
```
Why??? Ofcourse it fails, my proxy is in between. I said **NO** to Let's Encrypt in quick_jibri_installer.sh many times.

**Comment by R0GGER** [comment]
As a result of the Let's Encrypt error nginx fails to start, because it's trying to use a broken certificate...

```
root@recorder:/etc/jitsi/videobridge# systemctl status nginx.service
* nginx.service - A high performance web server and a reverse proxy server
   Loaded: loaded (/lib/systemd/system/nginx.service; enabled; vendor preset: enabled)
   Active: failed (Result: exit-code) since Thu 2021-01-28 11:14:42 UTC; 28s ago
     Docs: man:nginx(8)
  Process: 1275 ExecStartPre=/usr/sbin/nginx -t -q -g daemon on; master_process on; (code=exited, status=1/FAILURE)

Jan 28 11:14:42 recorder systemd[1]: Starting A high performance web server and a reverse proxy server...
Jan 28 11:14:42 recorder nginx[1275]: nginx: [emerg] BIO_new_file("/etc/letsencrypt/live/recorder.domain.nl/fullchain.pem") failed (SSL: err
Jan 28 11:14:42 recorder nginx[1275]: nginx: configuration file /etc/nginx/nginx.conf test failed
Jan 28 11:14:42 recorder systemd[1]: nginx.service: Control process exited, code=exited status=1
Jan 28 11:14:42 recorder systemd[1]: nginx.service: Failed with result 'exit-code'.
Jan 28 11:14:42 recorder systemd[1]: Failed to start A high performance web server and a reverse proxy server.
```

**Comment by Ark74** [comment]
Fixed on #29

**Comment by Ark74** [comment]
Please feel free to reopen if you find any issue.
Regards.
